mutiny-core/src/federation.rs

```diff
@@ -603,6 +603,29 @@ impl<S: MutinyStorage> FederationClient<S> {
         }
     }
 
+    pub(crate) async fn reissue(&self, oob_notes: OOBNotes) -> Result<bool, MutinyError> {
+        let logger = Arc::clone(&self.logger);
+
+        // Get the `MintClientModule`
+        let mint_module = self.fedimint_client.get_first_module::<MintClientModule>();
+
+        // TODO: (@leonardo) Do we need any `extra_meta` ?
+        // Reissue `OOBNotes`
+        let operation_id = mint_module.reissue_external_notes(oob_notes, ()).await?;
+
+        // TODO: (@leonardo) re-think about the results and errors that we need/want
+        match process_reissue_outcome(&mint_module, operation_id, logger.clone()).await? {
+            ReissueExternalNotesState::Created | ReissueExternalNotesState::Failed(_) => {
```

Comment:
Why return error on created

Reply:
I'm not sure how to proceed with this, but if the stream is already finished and the final result is still Created, it didn't get processed by the federation without a real guaranteeing it will.

Reply:
@benthecarman What do you think we should do here with Created variant?
It's not supposed to have that after the whole stream is consumed, so still handle as failure/error ? 🤔

Reply:
in `process_reissue_outcome` you only return the outcome if it is `Failed` or `Done` so you should just handle those two here, and leave the rest in the `_ => ` case where you can throw an error.

Reply:
wdyt about [eb59a8b](https://github.com/MutinyWallet/mutiny-node/pull/1037/commits/eb59a8b357aec7b89b7c7c487b2e2334185f0aa4)